<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XSFindDialog</class>
 <widget class="QDialog" name="XSFindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0">
   <item row="7" column="3">
    <widget class="QPushButton" name="btn_replace">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>107</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QGroupBox" name="gbox_option">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>144</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>144</height>
      </size>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QCheckBox" name="checkBox_case">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Case sensitive</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_wrap">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Wrap seach</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_whole">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Whole word</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_incremental">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Incremental</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_regular">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Regular expressions</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QGroupBox" name="gbox_scope">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>122</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>122</height>
      </size>
     </property>
     <property name="title">
      <string>Scope</string>
     </property>
     <widget class="QRadioButton" name="rbtn_all">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>All</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_selected">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected lines</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="btn_find">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>107</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>87</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="gbox_direction">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Direction</string>
     </property>
     <widget class="QRadioButton" name="rbtn_forward">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Forward</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_backword">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Backward</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Replace width:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="btn_replace_find">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>107</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Repalce/Find</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="btn_replace_all">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>107</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Replace All</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QComboBox" name="cbox_replace">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QComboBox" name="cbox_find">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_result">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbox_find</tabstop>
  <tabstop>cbox_replace</tabstop>
  <tabstop>rbtn_forward</tabstop>
  <tabstop>rbtn_backword</tabstop>
  <tabstop>rbtn_all</tabstop>
  <tabstop>rbtn_selected</tabstop>
  <tabstop>checkBox_case</tabstop>
  <tabstop>checkBox_whole</tabstop>
  <tabstop>checkBox_regular</tabstop>
  <tabstop>checkBox_wrap</tabstop>
  <tabstop>checkBox_incremental</tabstop>
  <tabstop>btn_find</tabstop>
  <tabstop>btn_replace_find</tabstop>
  <tabstop>btn_replace</tabstop>
  <tabstop>btn_replace_all</tabstop>
  <tabstop>btn_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
